File: SonicXBackendC/SonicXBackendC/MusicManagment/MusicFile.cs
using System;

namespace SonicXBackendC.MusicManagment
{
    public class MusicFile
    {
        private string title;
        private string artist;
        private string album;
        private int duration;
        private string path;

        public MusicFile(string title, string artist, string album, int duration, string path)
        {
            this.title = title;
            this.artist = artist;
            this.album = album;
            this.duration = duration;
            this.path = path;
        }

        public string Title
        {
            get { return title; }
            set { title = value; }
        }

        public string Artist
        {
            get { return artist; }
            set { artist = value; }
        }

        public string Album
        {
            get { return album; }
            set { album = value; }
        }

        public int Duration
        {
            get { return duration; }
            set { duration = value; }
        }

        public string Path
        {
            get { return path; }
            set { path = value; }
        }

        public override string ToString()
        {
            return $"MusicFile{{title='{title}', artist='{artist}', album='{album}', duration={duration}, path='{path}'}}";
        }
    }
}

File: SonicXBackned/src/main/java/org/example/MusicManagment/MusicFile.java
package org.example.MusicManagment;

public class MusicFile {
    private String title;
    private String artist;
    private String album;
    private int duration;
    private String path;

    public MusicFile( String title, String artist, String album, int duration, String path) {
        this.title = title;
        this.artist = artist;
        this.album = album;
        this.duration = duration;
        this.path = path;
    }




    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getArtist() {
        return artist;
    }

    public void setArtist(String artist) {
        this.artist = artist;
    }

    public String getAlbum() {
        return album;
    }

    public void setAlbum(String album) {
        this.album = album;
    }

    public int getDuration() {
        return duration;
    }

    public void setDuration(int duration) {
        this.duration = duration;
    }

    public String getPath() {
        return path;
    }

    public void setPath(String path) {
        this.path = path;
    }

    @Override
    public String toString() {
        return "MusicFile{" +
                ", title='" + title + '\'' +
                ", artist='" + artist + '\'' +
                ", album='" + album + '\'' +
                ", duration=" + duration +
                ", path='" + path + '\'' +
                '}';
    }
}
